crypto/ethsecp256k1/ethsecp256k1.go

```diff
@@ -3,17 +3,25 @@ package ethsecp256k1
 import (
 	"bytes"
 	"crypto/ecdsa"
+	"crypto/subtle"
+	"fmt"
 
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
 	tmcrypto "github.com/tendermint/tendermint/crypto"
 )
 
 const (
 	// PrivKeySize defines the size of the PrivKey bytes
 	PrivKeySize = 32
-	// KeyType is the string constant for the EthSecp256k1 algorithm
+	// PubKeySize defines the size of the PubKey bytes
+	PubKeySize = 33
```

Comment:
i think public keys in ethereum are 64 bytes. is 33 correct here?

Reply:
the compressed key is 33 bytes